playbooks/hypervisor_deploy_subscriber_tests.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
+---
+- hosts: hypervisors
+  name: Subscriber tests
+  vars:
+    - svtools_archive_commit_id: 0c7b539
+    - poll_timeout: 5
+    - docker_test_image_name: hyp_sub_tests
+  tasks:
+  - name: Copy svtools archive on target
+    copy:
+      src: ../src/svtools/svtools_{{ svtools_archive_commit_id }}.tar
+      dest: /tmp/
+  - name: Load svtools as Docker image
+    shell:
```

Comment:
Remove that shell line.
You can just write
`cmd : docker load --input /tmp/svtools_{{ svtools_archive_commit_id }}.tar`

shell is only needed for working with environnement variables or for operations (<, >, |, & ... )

correct all other shell occurence as well